src/System.Memory/ref/System.Memory.cs

```diff
@@ -200,6 +214,12 @@ public static partial class MemoryExtensions
 }
 namespace System.Buffers
 {
+    public partial interface IMemoryList<T>
```

Comment:
Is there a "ReadOnly" equivalent of this?

Reply:
This also came up in the meeting https://youtu.be/tLnQsNTfl9Q